File: Homeworks/JSFundamentals/ConditionalStatements/quadraticEquation.js
console.log(solve([5, 2, 8]));
function solve(array) {
    var a = parseFloat(array[0]);
    var b = parseFloat(array[1]);
    var c = parseFloat(array[2]);
    var diskriminant = Math.sqrt(Math.pow(b, 2) - 4*a*c);

    var root1 = (-b + diskriminant) / (2 * a);

    var root2 = (-b - diskriminant) / (2 * a);
    if(diskriminant == 0){
        return "x1=x2=" + parseFloat(Math.round((-b / (2 * a) ) * 100 / 100)).toFixed(2);
    }
    if (isNaN(root1) && isNaN(root2)) {
        return 'no real roots';
    } else {
        if(root1 < root2){
            return "x1=" + parseFloat(Math.round(root1 * 100) / 100).toFixed(2) + "; x2=" + parseFloat(Math.round(root2 * 100) / 100).toFixed(2);
        }else {
            return "x1=" + parseFloat(Math.round(root2 * 100) / 100).toFixed(2) + "; x2=" + parseFloat(Math.round(root1 * 100) / 100).toFixed(2);
        }
    }
}

File: Homeworks/C#1/ConsoleInputOutput/06.QuadraticEquation/QuadraticEquation.cs
using System;

class QuadraticEquation
{
    static void Main(string[] args)
    {
        double a = double.Parse(Console.ReadLine());
        double b = double.Parse(Console.ReadLine());
        double c = double.Parse(Console.ReadLine());
        double disrminant = (b * b) - (4 * (a * c));
        double root1 = (-b - Math.Sqrt(disrminant)) / (2 * a);
        double root2 = (-b + Math.Sqrt(disrminant)) / (2 * a);
        if (disrminant < 0)
        {
            Console.WriteLine("no real roots");
        }
        else
        {
            if (root1 == root2)
            {
                Console.WriteLine("{0:0.00}", root1);
            }
            else
            {
                Console.WriteLine("{0:0.00}", root1);
                Console.WriteLine("{0:0.00}", root2);
            }
            
            
        }
    }
}

